documentation/tutorials/batch.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: documentation
+menu: tut_batch
+subtitle: 'Tutorial: Batch'
+redirect_from:
+- /getStarted/batch/
+- /documentation/getstarted/batch/
+- /documentation/getstarted/advanced/batch/
```

Comment:
New files won't need redirects

Redirects were set up when we restructured the website but new files don't have that problem

Reply:
9bd2ec3b5bbf53c9fbab0b5c5fc35615d5dc8144